packages/bpk-component-horizontal-nav/src/BpkHorizontalNavItem.js

```diff
@@ -55,7 +55,7 @@ const BpkHorizontalNavItem = props => {
     <button
       type="button"
       className={innerClassNames.join(' ')}
-      disabled={selected}
+      aria-disabled={selected}
```

Comment:
Necessary to remove the `disabled` state for this as it prevents the grab scrolling events from firing.

Unfortunately this will probably be a problem with anything else disabled that gets wrapped in a mobile scroll container.